zerver/lib/upload.py

```diff
@@ -353,6 +353,9 @@ def upload_avatar_image(self, user_file, user_profile, email):
         resized_medium = resize_avatar(image_data, MEDIUM_AVATAR_SIZE)
         write_local_file('avatars', email_hash+'-medium.png', resized_medium)
 
+        user_profile.avatar_version += 1
```

Comment:
So in reality we are treating bots and the users as different entities and writing the code accordingly, right?